app/src/main/java/org/oppia/app/testing/FontScaleTestActivity.kt

```diff
@@ -0,0 +1,29 @@
+package org.oppia.app.testing
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.utility.FontScaleConfigurationUtil
+
+private const val FONT_SCALE_EXTRA_KEY = "FONT_SCALE_EXTRA_KEY"
+
+/** Test Activity used for testing Font Scale */
```

Comment:
Nit: `/** Test activity used for testing font scale. */`

Reply:
updated as per suggestion.